tests/GenerateTestCompares/CXSM.cpp

```diff
@@ -44,99 +48,99 @@ Compare_CXSM::Compare_CXSM()
   CheckTripleTree.at(0).at(5).at(0) = 10.5468;
-  CheckTripleCT.at(0).at(5).at(0) = -0.107203;
-  CheckTripleCW.at(0).at(5).at(0) = 0.107203;
+  CheckTripleCT.at(0).at(5).at(0)   = -0.107203;
+  CheckTripleCW.at(0).at(5).at(0)   = 0.107203;
   CheckTripleTree.at(1).at(1).at(4) = -63.5495;
-  CheckTripleCT.at(1).at(1).at(4) = 5.21565;
-  CheckTripleCW.at(1).at(1).at(4) = -5.21565;
+  CheckTripleCT.at(1).at(1).at(4)   = 5.21565;
+  CheckTripleCW.at(1).at(1).at(4)   = -5.21565;
   CheckTripleTree.at(1).at(1).at(5) = 10.5468;
-  CheckTripleCT.at(1).at(1).at(5) = -0.107203;
-  CheckTripleCW.at(1).at(1).at(5) = 0.107203;
+  CheckTripleCT.at(1).at(1).at(5)   = -0.107203;
+  CheckTripleCW.at(1).at(1).at(5)   = 0.107203;
   CheckTripleTree.at(1).at(4).at(1) = -63.5495;
-  CheckTripleCT.at(1).at(4).at(1) = 5.21565;
-  CheckTripleCW.at(1).at(4).at(1) = -5.21565;
+  CheckTripleCT.at(1).at(4).at(1)   = 5.21565;
+  CheckTripleCW.at(1).at(4).at(1)   = -5.21565;
   CheckTripleTree.at(1).at(5).at(1) = 10.5468;
-  CheckTripleCT.at(1).at(5).at(1) = -0.107203;
-  CheckTripleCW.at(1).at(5).at(1) = 0.107203;
+  CheckTripleCT.at(1).at(5).at(1)   = -0.107203;
+  CheckTripleCW.at(1).at(5).at(1)   = 0.107203;
   CheckTripleTree.at(2).at(2).at(4) = -63.5495;
-  CheckTripleCT.at(2).at(2).at(4) = 5.21565;
-  CheckTripleCW.at(2).at(2).at(4) = -5.21565;
+  CheckTripleCT.at(2).at(2).at(4)   = 5.21565;
+  CheckTripleCW.at(2).at(2).at(4)   = -5.21565;
   CheckTripleTree.at(2).at(2).at(5) = 10.5468;
-  CheckTripleCT.at(2).at(2).at(5) = -0.107203;
-  CheckTripleCW.at(2).at(2).at(5) = 0.107203;
+  CheckTripleCT.at(2).at(2).at(5)   = -0.107203;
+  CheckTripleCW.at(2).at(2).at(5)   = 0.107203;
   CheckTripleTree.at(2).at(4).at(2) = -63.5495;
-  CheckTripleCT.at(2).at(4).at(2) = 5.21565;
-  CheckTripleCW.at(2).at(4).at(2) = -5.21565;
+  CheckTripleCT.at(2).at(4).at(2)   = 5.21565;
+  CheckTripleCW.at(2).at(4).at(2)   = -5.21565;
   CheckTripleTree.at(2).at(5).at(2) = 10.5468;
-  CheckTripleCT.at(2).at(5).at(2) = -0.107203;
-  CheckTripleCW.at(2).at(5).at(2) = 0.107203;
+  CheckTripleCT.at(2).at(5).at(2)   = -0.107203;
+  CheckTripleCW.at(2).at(5).at(2)   = 0.107203;
   CheckTripleTree.at(3).at(3).at(4) = -0.200227;
-  CheckTripleCT.at(3).at(3).at(4) = 0.23385;
-  CheckTripleCW.at(3).at(3).at(4) = -0.234681;
+  CheckTripleCT.at(3).at(3).at(4)   = 0.23385;
+  CheckTripleCW.at(3).at(3).at(4)   = -0.234681;
   CheckTripleTree.at(3).at(3).at(5) = -694.62;
-  CheckTripleCT.at(3).at(3).at(5) = 38.4946;
-  CheckTripleCW.at(3).at(3).at(5) = -38.8638;
+  CheckTripleCT.at(3).at(3).at(5)   = 38.4946;
+  CheckTripleCW.at(3).at(3).at(5)   = -38.8638;
   CheckTripleTree.at(3).at(4).at(3) = -0.200227;
-  CheckTripleCT.at(3).at(4).at(3) = 0.23385;
-  CheckTripleCW.at(3).at(4).at(3) = -0.234681;
+  CheckTripleCT.at(3).at(4).at(3)   = 0.23385;
+  CheckTripleCW.at(3).at(4).at(3)   = -0.234681;
   CheckTripleTree.at(3).at(5).at(3) = -694.62;
-  CheckTripleCT.at(3).at(5).at(3) = 38.4946;
```

Comment:
clang changed? Or why do we have this as commit :D?

Reply:
These are generated by https://github.com/phbasler/BSMPT-UnitTestReferenceCreator. Seems like clang formatting adjusted them